source/model/backtraceMetrics.ts

```diff
@@ -201,14 +203,6 @@ export class BacktraceMetrics {
     return localStorage.getItem('sessionId') || undefined;
   }
 
-  /**
-   * Get stored time since last session created or current time
-   */
-  private getSessionStart(): number | undefined {
```

Comment:
why we removed this code?

Reply:
It wasn't being used for anything. The data could be used to get session length, but for now it was an unused function